src/frontend-admin/src/components/App/App.module.css

```diff
@@ -16,3 +16,105 @@
 .appLink {
   color: #61dafb;
 }
+
+.navContainer {
+  background: white;
+  height: 140px;
+}
+
+nav {
+  width: 100%;
+  height: 68px;
+  background: #f4f4f4;
+
+  display: flex;
+  flex-direction: row;  
+  justify-content: flex-end;
+  z-index: 3;
+}
+
+nav h1 {
+  flex-grow: 1;
+  align-self: flex-start;
+  margin: auto;
+  margin-left: 100px;
+  
+  
+  color: #000000;	
+  /*font-family: CQL;	*/
```

Comment:
Good call on this one - I recommend adding a comment line above this one to the effect of: TODO: uncomment when cql fonts are added to the project.